client/src/services/DownloadPdf.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import axios from "axios";
+
+const BACKEND_URL = process.env.REACT_APP_BACKEND_URL;
+const API_URL = BACKEND_URL + "/api/interview";
+
+const DownloadPdf = () => {
+  const handleDownload = async () => {
+    try {
+      const interviewId = "6710d080fbc135de4b73978e";
```

Comment:
Don't hardcode the interviewId. Fetch based on the user and the particular job from the database.

Reply:
Yaa it is not implemented yet. Just I tested it. If the say me where to integrate it I will integrate it there, and I will remove the hardcode

Reply:
But I will remove the hardcode now and I will push it again